templates/controllers/modals/editorDecision/form/promoteForm.tpl

```diff
@@ -60,6 +60,18 @@
 					</a>
 				{/fbvFormSection}
 			{/if}
+
```

Comment:
Since the two template changes appear to be identical, could you wrap them up in a separate .tpl file and include them in each place they're used?

Reply:
I was thinking about it, but as I didn't know the minimum requirement for creating an include, I just duplicated it :)